tests/bootstrap.php

```diff
@@ -2,17 +2,15 @@
 /**
  * @author Eugene Terentev <eugene@terentev.net>
  */
-// Composer
-require __DIR__ . '/../vendor/autoload.php';
-
-// Set environment
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'test');
-defined('YII_APP_BASE_PATH') or define('YII_APP_BASE_PATH', dirname(__DIR__));
 
 // Environment
 $dotenv = new \Dotenv\Dotenv(dirname(__DIR__));
 $dotenv->load();
 $dotenv->required('TEST_DB_DSN');
 $dotenv->required('TEST_DB_USERNAME');
 $dotenv->required('TEST_DB_PASSWORD');
+
+// Set environment
+defined('YII_DEBUG') or define('YII_DEBUG', true);
+defined('YII_ENV') or define('YII_ENV', env('YII_ENV', 'test'));
```

Comment:
I found the problem is actually here.

You're using the `env()` function while it's not defined.

Reply:
Dear, @XzAeRo.
Problem is already solved. 
`env()` function is defined in common/helpers.php file which i included in frontend/web/index-test.php, not in tests/bootstrap.php. We could re-arrange `require` statements if you wish. You're the boss.

Reply:
I like your current suggestion. I will keep an eye on the tests since they keep failing for some reason. Thanks!

Reply:
Would you mind opening #658 again ?